winml/lib/Api.Core/WinMLAdapter.cpp

```diff
@@ -297,6 +318,8 @@ class WinMLAdapter : public Microsoft::WRL::RuntimeClass<
 
  public:
   WinMLAdapter() : lotus_environment_(PheonixSingleton<WinML::LotusEnvironment>()) {
+    // register ETW manager on adapter construction
+    static WinMLTraceLoggingProviderManager etw_manager;
```

Comment:
Should we catch the possible runtime_error from its constructor or just let it through?

Reply:
Got rid of the class and added into lotus environment :)